integration-cli/docker_cli_run_test.go

```diff
@@ -106,6 +106,33 @@ func TestRunEchoStdoutWithCPUAndMemoryLimit(t *testing.T) {
 }
 
 // "test" should be printed
+func TestRunEchoStdoutWitCPUQuota(t *testing.T) {
+	defer deleteAllContainers()
+
+	runCmd := exec.Command(dockerBinary, "run", "--cpu-quota", "8000", "--name", "test", "busybox", "echo", "test")
+	out, _, _, err := runCommandWithStdoutStderr(runCmd)
+	if err != nil {
+		t.Fatalf("failed to run container: %v, output: %q", err, out)
+	}
+	out = strings.TrimSpace(out)
+	if out != "test" {
```

Comment:
@cpuguy83  This test failure on quota not supported machine is because the `out` is `
WARNING: Your kernel does not support CPU cfs quota. Quota discarded.
test`, So I think if the `out` contains `WARNING: Your kernel does not support CPU cfs quota. Quota discarded.` 
we can skip this test. WDYT?